### PR TITLE
FromThePage IIIF API support, v2

### PR DESCRIPTION
**Pull request by benwbrum**
Implement the second revision of support for IIIF, including IIIF-based API access to all contributions made for a project within a time window.

### CODE REVIEW

File: app/models/deed.rb [hunk 1]
```diff
@@ -13,6 +13,21 @@ class Deed < ActiveRecord::Base
...
+    NOTE_ADDED,
+    PAGE_TRANSLATED,
+    PAGE_TRANSLATION_EDIT,
+    OCR_CORRECTED,
+    NEEDS_REVIEW,
+    TRANSLATION_REVIEW,
+    TRANSLATION_INDEXED
+  ]
+
+
```
Comment (saracarl) [inline app/models/deed.rb]: Didn't we already have CONTRIBUTOR_DEED_TYPES ?  Why did we have to add it here?

File: app/controllers/export_controller.rb [hunk 2]
```diff
@@ -94,6 +94,48 @@ def export_all_works
...
+  end
+
+  def work_plaintext_translation_emended
+    render  :layout => false, :content_type => "text/plain", :text => @work.emended_translation_plaintext
+  end
+
+  def work_plaintext_searchable
+    render  :layout => false, :content_type => "text/plain", :text => @work.searchable_plaintext
+  end
+
```
Comment (saracarl) [inline app/controllers/export_controller.rb]: Please open issues to consider adding these extra export formats to the Export tab.

File: app/models/work_statistic.rb [hunk 3]
```diff
@@ -55,6 +55,23 @@ def pct_translation_blank
...
+  def pct_transcribed_or_blank
+    pct_blank + pct_transcribed
+  end
+  
+  def pct_translated_or_blank
+    pct_translation_blank + pct_translated    
+  end
+  
+  def pct_transcribed_or_blank_not_review
+
```
Comment (saracarl) [inline app/models/work_statistic.rb]: Um.  This is a blank method?

File: app/models/work_statistic.rb [hunk 4]
```diff
@@ -55,6 +55,23 @@ def pct_translation_blank
...
+  def pct_translated_or_blank
+    pct_translation_blank + pct_translated    
+  end
+  
+  def pct_transcribed_or_blank_not_review
+    
+  end
+  
+  def pct_translated_or_blank_not_review
+
```
Comment (saracarl) [inline app/models/work_statistic.rb]: This is a blank method?

File: app/controllers/export_controller.rb (hunk 2 above)
Comment (benwbrum) [reply app/controllers/export_controller.rb]: Done -- see #898

File: app/models/deed.rb (hunk 1 above)
Comment (benwbrum) [reply app/models/deed.rb]: Not that I was able to find.  Where have you seen it?

Comment (tbl73) [reply app/models/deed.rb]: I don't think we have contributor_deed_types, but deeds are already associated with user_ids.  What are these new contributor_deed_types used for?

**Pull request merged**